api/openapi-spec/v3/api__v1_openapi.json

```
@@ -1292,6 +1305,26 @@
         ],
         "type": "object"
       },
+      "io.k8s.api.core.v1.ContainerResizePolicy": {
+        "description": "ContainerResizePolicy represents resource resize policy for a single container.",
+        "properties": {
+          "policy": {
+            "default": "",
+            "description": "Resource resize policy applicable to the above mentioned resource. If not specified, it defaults to RestartNotRequired.",
```

"above mentioned" is not ideal here as in this file it was generated "below". Maybe say:
```suggestion
            "description": "Resource resize policy applicable to the resource specified by field resourceName. If policy is not specified, it defaults to RestartNotRequired.",
```